app/src/main/java/com/onboarding/parkingsystemkotlin/mvp/view/ConfigureParkingView.kt

```diff
@@ -0,0 +1,33 @@
+package com.onboarding.parkingsystemkotlin.mvp.view
+
+import android.widget.Toast
+import androidx.fragment.app.DialogFragment
+import com.onboarding.parkingsystemkotlin.R
+import com.onboarding.parkingsystemkotlin.databinding.ConfigureParkingLotsDialogFragmentMainActivityBinding
+import com.onboarding.parkingsystemkotlin.mvp.contract.ConfigureParkingLotsContract
+import com.onboarding.parkingsystemkotlin.mvp.view.base.FragmentView
+
+class ConfigureParkingView(
+    fragment: DialogFragment,
+    private val binding: ConfigureParkingLotsDialogFragmentMainActivityBinding
+) : FragmentView(fragment), ConfigureParkingLotsContract.ConfigureParkingView {
+
+    override fun getLots(): String = binding.editTextDialogFragmentParkingInput.text.toString()
+
+    override fun closeDialog() {
+        (getFragment() as DialogFragment?)?.dismiss()
+    }
+
+    override fun showEmptyInputToast() {
```

Comment:
you can unify the toast method and send the message in the parameter

Reply:
done